.github/workflows/ci.yaml

```diff
@@ -181,3 +181,52 @@ jobs:
                 env:
                     DATABASE_URL: "mysql://mysql:mysql@127.0.0.1:${{ job.services.mysql.ports[3306] }}/testdb"
                     DATABASE: "mysql://mysql:mysql@127.0.0.1:${{ job.services.mysql.ports[3306] }}/testdb"
+
+    solr-integration:
+        name: "Solr integration tests"
+        runs-on: "ubuntu-20.04"
+        timeout-minutes: 30
+        permissions:
+            packages: read
+            contents: read
+        strategy:
+            fail-fast: false
+            matrix:
+                php:
+                    - '7.4'
+                    - '8.0'
+                    - '8.1'
+        steps:
+            -   name: Start redis container
+                run: >- 
+                    docker run 
+                    --name redis 
+                    -p 6379:6379 
+                    -d 
+                    --entrypoint redis-server redis --appendfsync no --maxmemory 60mb --save ""
```

Comment:
Why was redis container not declared as a GH action service?

See integration tests in product catalog declare postgres as a service:
```yaml
    integration-tests-postgres:
        name: PostgreSQL integration tests
        needs: tests
        services:
            postgres:
                image: postgres:10
                ports:
                    - 5432
                env:
                    POSTGRES_PASSWORD: postgres
                    POSTGRES_DB: testdb
                options: >-
                    --health-cmd pg_isready
                    --health-interval 10s
                    --health-timeout 5s
                    --health-retries 5
                    --tmpfs /var/lib/postgresql/data
```

Reply:
I guess I could try doing something similar to https://github.community/t/how-do-i-properly-override-a-service-entrypoint/17435/8 (create a redis service and configure it in a separate step), what do you think?

Reply:
The same options can be specified in a configuration file instead. We could simply provide an image with added layer, where this config is added.

If necessary, the same configuration file can be mounted as a service volume: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idservicesservice_idvolumes